### DISCUSSION TITLE
Extend login token duration

### DISCUSSION THREAD

**Discussion by masterjuggler** [Difegue/LANraragi#965] [Q&A]
Is there any way to extend how long it takes before I need to log back into lanraragi again?

It happens frequently that I'll come back to a gallery I had open and have it error out when switching pages. Currently this isn't handled gracefully, so I need to refresh the page to pull up the login screen.

**Comment by Difegue** [comment]
Just edit the login cookie's length to be 60 years or something -- I wouldn't expect it to expire that quickly however, something weird might be going on?

**Reply by masterjuggler** [reply]
How exactly do you go about editing the cookie's login length? I haven't done that before.

I haven't timed exactly how long I stay logged in for, but it's definitely not more than a week.
